collector/mountstats_linux.go

```diff
@@ -111,8 +138,8 @@ func NewMountStatsCollector() (Collector, error) {
 	)
 
 	var (
-		labels   = []string{"export"}
-		opLabels = []string{"export", "operation"}
+		labels   = []string{"export", "mountpoint", "xprt"}
```

Comment:
According to [this doc](http://www.fsl.cs.stonybrook.edu/~mchen/mountstat-format.txt), the first column of the `xprt` line is `srcport`, so the label should be `srcport` not `xprt`.

Reply:
No problem. I will update this to ```srcport```. I will also update ```collector/fixtures/e2e-64k-page-output.txt```.